core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -159,6 +159,26 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers wit
     }
   }
 
+  it should "properly check For existing entity types" in withDefaultTestDatabase {
+    withWorkspaceContext(testData.workspace) { context =>
+      val pair2EntityTypeExists = runAndWait(entityQuery.doesEntityTypeAlreadyExist(context, "Pair2")).head
+      assert(!pair2EntityTypeExists)
+      val pairEntityTypeExists = runAndWait(entityQuery.doesEntityTypeAlreadyExist(context, "Pair")).head
+      assert(pairEntityTypeExists)
+    }
+  }
+
+  it should "change entity type name" in withDefaultTestDatabase {
+    withWorkspaceContext(testData.workspace) { context =>
+      val pair2EntityTypeExists = runAndWait(entityQuery.doesEntityTypeAlreadyExist(context, "Pair2")).head
+      assert(!pair2EntityTypeExists)
+      runAndWait(entityQuery.changeEntityTypeName(context, "Pair", "Pair2"))
+      val pair2EntityTypeExistsAfterUpdate = runAndWait(entityQuery.doesEntityTypeAlreadyExist(context, "Pair2")).head
+      assert(pair2EntityTypeExistsAfterUpdate)
+
+    }
+  }
+
```

Comment:
we've bemoaned how long Rawls unit tests take to run … a big contributor to that runtime is that each unit test truncates all tables and then repopulates with the exemplar data it needs for the test. `withDefaultTestDatabase` is a large exemplar dataset. Is there a smaller exemplar dataset you can use? Check out `withMinimalTestDatabase` instead (I don't know if it has any entity data), or `withConstantTestDatabase`, or maybe there's something else in there …

Reply:
I changed one test (where we verify that an entity type does not exist) to `withMinimalTestDatabase` and that saves 800ms in my testing. Unfortunately, the minimal data set doesn't have entities so I left the other tests using the the default.

Reply:
I know 800ms seems trivial, but we're currently at 1808 unit tests. 1808 * 800ms ≈ 24 minutes … gotta do everything we can!

Reply:
Totally fair, I mentioned it because I was surprised it was that much.  Test went from 1.9s to 1.1s.